web_ext/rust_webserver/Cargo.toml

```diff
@@ -7,15 +7,15 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-algebra = { path = "../ext/crates/algebra", default-features = false }
-bivec = { path = "../ext/crates/bivec", features = ["json"] }
-chart = { path = "../ext/crates/chart" }
-error = { path = "../ext/crates/error" }
-ext = { path = "../ext", default-features = false }
-fp = { path = "../ext/crates/fp", features = ["json"], default-features = false }
-once = { path = "../ext/crates/once" }
-sseq = { path = "../ext/crates/sseq" }
-thread-token = { path = "../ext/crates/thread-token", optional = true }
+algebra = { path = "../../ext/crates/algebra", default-features = false }
```

Comment:
It probably makes sense to reexport these from the `ext` crate, which means we only have one places where we fuss around with dependencies for the helper crates. That will probably be more work and worth punting to a followup, since we'd need to start writing `ext::fp` and whatnot everwhere.

There will likely be some work to sort out all the cargo features in a coherent fashion...

Reply:
Does it really matter, though? The project is small enough that build times would not be particularly affected. I plan to do a pass over the subcrates to start documenting them, though, so maybe we can see if it makes sense to re-export some things or not as submodules of `ext` after I do that.

Reply:
Got it. I'm happy to investigate a way to do improve the interface in the future. Like I said, I want to focus on writing docs first.